services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx

```diff
@@ -392,13 +408,30 @@ export const SubmissionType = ({
                             />
                         </fieldset>
                         <PageActions
-                            pageVariant="FIRST"
+                            pageVariant={
+                                isNewSubmission ? 'FIRST' : 'EDIT-FIRST'
+                            }
                             backOnClick={() => history.push('/dashboard')}
                             continueOnClick={() => {
                                 setShouldValidate(true)
                                 setFocusErrorSummaryHeading(true)
                             }}
-                            saveAsDraftOnClick={() => setShouldValidate(true)}
+                            saveAsDraftOnClick={async () => {
+                                setShouldValidate(true)
+                                //Validate form before trying to save a draft
+                                const validationErrors = await validateForm(
+                                    values
+                                )
+                                if (
+                                    Object.keys(validationErrors).length === 0
+                                ) {
+                                    await handleFormSubmit(
+                                        values,
+                                        { setSubmitting },
+                                        { redirectPath: '/dashboard' }
```

Comment:
Passing relevant arguments for the `handleFormSubmit`

Reply:
This all seems solid, following the pattern of the other pages. ContractDetails just passes setSubmitting directly instead of wrapping it in an object

Reply:
+1 We can pass `setSubmitting` directly here as well just like ContractDetails. No need to pass all the formik state, we aren't using it.